spec/system/candidate_interface/candidate_viewing_application_spec.rb

```diff
@@ -7,6 +7,11 @@
     given_i_am_signed_in
     when_i_visit_the_site
     then_i_should_see_that_i_have_made_no_choices
+    and_i_should_not_see_the_covid19_banner
+
+    when_covid19_feature_flag_is_active
+    and_i_visit_the_site
```

Comment:
Nitpick, feel free to ignore!

```suggestion
    given_covid19_feature_flag_is_active
    when_i_visit_the_site
```